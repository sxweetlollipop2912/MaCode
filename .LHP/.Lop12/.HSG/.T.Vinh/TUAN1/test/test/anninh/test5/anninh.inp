18 25
447 2650
682 4676
1054 1339
1144 1827
1195 2604
1406 3332
1938 4682
2152 1909
2262 4873
2662 2731
3099 2910
3211 1499
3944 191
4000 3313
4039 2218
4537 2062
4656 4354
4802 3402
1850 1212
3275 1314
156 3392
754 3767
1354 3153
4020 883
1210 3628
4628 2899
3759 1992
131 1971
3081 1779
2051 2303
1535 2391
1977 1961
49 3
2050 2035
4280 1974
1927 3082
4110 1504
2885 3457
3832 3814
3999 3705
3310 4609
3665 1129
1887 104
