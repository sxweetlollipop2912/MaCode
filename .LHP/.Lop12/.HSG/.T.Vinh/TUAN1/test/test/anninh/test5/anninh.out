697
185
742
652
179
692
296
503
280
679
280
394
340
175
1336
126
244
521
558
547
501
392
1048
454
1235
